### DISCUSSION TITLE
422 Error with Make.com when creating invoice

### DISCUSSION THREAD

**Discussion by franz-ctrl** [fattureincloud/api#373] [Q&A] [answered]
Hi! I'm trying to use Make.com with Fatture in cloud to create an invoice but I'm always getting this 422 error. Has someone some experience with integrating make.com with fatture in cloud and give some advice? thanks!

**Comment by valmoz** [comment]
Hi @franz-ctrl 
it is possible (but not certain) that you're experiencing a 422 error on Invoice Totals ("Il totale dei pagamenti non corrisponde al totale da pagare"), you could try to read this guide that may help you (even if it is not Make-related): https://developers.fattureincloud.it/docs/guides/invoice-totals

If you want more details about the error you're experiencing, unfortunately, you must contact our Customer Support on the Fatture in Cloud page. This way we'll have access to your Company ID and to the logs of your requests. These are details that we cannot share in this public discussion.

You can also try to contact Make's support, they should be able to guide you better on the usage of their software. From the Fatture in Cloud side, we can only give support on how the request should be built, but in this case, it's Make's role to actually build it. We don't actually have access to the Make dashboard or code.

I'll keep this conversation open just in case other Make.com users want to share their experiences.

Thanks
Mauro

**Reply by franz-ctrl** [reply, marked as answer]
Hi Mauro, thanks for your support on this. In the meantime I got in touch with Make and Fatture in cloud. Fatture weren't really helpful whereas Make advised that the error comes from the date format whereby Fatture requires to receive "Y-M-D". With this I was able to solve that specific error and could successfully send invoices. Now I'm just fixing another error whereby a payment status is getting incorrectly submitted to Fatture. Anyway, slowly getting there..
Thanks and regards,
Franz
